scripts/yocto-build-and-test.sh

```diff
@@ -213,18 +213,18 @@ PREFERRED_PROVIDER_virtual/mender-client-native = "mender-native"
 EOF
     fi
 
-    # Assuming sumo or newer
     cat >> $BUILDDIR/conf/local.conf <<EOF
-# MEN-2948: Renamed mender recipe -> mender-client
-# But the "mender" reference has to be kept for backwards compatibility
-# with 2.1.x, 2.2.x, and 2.3.x
```

Comment:
If you are removing this comment, also remove `pn-mender`.

Reply:
@kacf But isn't it needed for the new recipes in C++? That was my whole idea of removing the comment, that we still need it but for a different reason. Am I wrong?

Reply:
No, we will not call the C++ client `mender`, we will call it `mender-update`, most likely.

Reply:
Discussed over Slack, we need that to stay.